kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -28,52 +28,74 @@ public void beforeEveryTest() {
 
         @Test
         void addFigureTest() {
+            //Given
             ShapeCollector shapeCollector = new ShapeCollector();
             Triangle triangle = new Triangle(3, 6);
-            shapeCollector.addFigure(triangle);
             Circle circle = new Circle(10);
-            shapeCollector.addFigure(circle);
             Square square = new Square(2);
+
+            //When
+            shapeCollector.addFigure(triangle);
+            shapeCollector.addFigure(circle);
             shapeCollector.addFigure(square);
-            Assertions.assertEquals(3, shapeCollector.shapes.size());
+
+            //Then
+            Assertions.assertEquals(3, shapeCollector.shapes().size());
```

Comment:
ta asercja nie sprawdza tego, co trzeba - sprawdza tylko, ze są trzy figury w kolekcji, a to za malo. Musisz sprawdzic, ze w kolekcji sa trzy elementy oraz, ze ta kolekcja zawiera dokladnie Trinagle(3,6), Circle(10) oraz Square(2)